tests/app/dao/notification_dao/test_notification_dao_delete_notifications.py

```diff
@@ -157,28 +156,24 @@ def test_delete_notifications_inserts_notification_history(sample_service):
     assert NotificationHistory.query.count() == 2
 
 
-def test_delete_notifications_updates_notification_history(sample_email_template, mocker):
+def test_delete_notifications_does_nothing_if_notification_history_if_row_already_exists(
```

Comment:
Optional: This test name doesn't quite read correctly, should maybe be `test_delete_notifications_does_nothing_if_notification_already_exists_in_notification_history`